Documentation/ColumnsConfig/Type/Password/Properties/PasswordGenerator.rst

```diff
@@ -197,53 +196,53 @@ passwordRules.random
         `passwordRules.random` is set to one
         of the available encodings: :php:`hex` or :php:`base64`.
 
-.. _columns-password-properties-passwordGenerator_passwordRules_digitcharacters:
+..  _columns-password-properties-passwordGenerator_passwordRules_digitcharacters:
 
 passwordRules.digitCharacters
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ..  confval:: passwordRules.digitCharacters
-
+    :name: passwordRules-digitCharacters
```

Comment:
Would you want to always repeat the password part? `passwordRules-digitCharacters` should always be unique?

Reply:
I like consistency, otherwise you have to think about it when to use password as prefix and when not. Additionally, this way, it is not really obvious, that this is part of the password type.